components/brave_ads/common/custom_notification_ads_features.h

```diff
@@ -0,0 +1,89 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_COMMON_CUSTOM_NOTIFICATION_ADS_FEATURES_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_COMMON_CUSTOM_NOTIFICATION_ADS_FEATURES_H_
+
+#include <string>
+
+#include "base/feature_list.h"
+#include "base/metrics/field_trial_params.h"
+#include "brave/components/brave_ads/common/custom_notification_ads_constants.h"
+#include "build/build_config.h"
+
+namespace brave_ads::features {
+
+BASE_DECLARE_FEATURE(kCustomNotificationAds);
+BASE_DECLARE_FEATURE(kAllowedToFallbackToCustomNotificationAds);
+
+bool IsCustomNotificationAdsEnabled();
+
+bool IsAllowedToFallbackToCustomNotificationAdsEnabled();
+
+// Set to true to fallback to custom notification ads if native notifications
+// are disabled or false to never fallback
+constexpr base::FeatureParam<bool> kCanFallbackToCustomNotificationAds{
+    &kCustomNotificationAds, "can_fallback_to_custom_notifications",
+    kDefaultCanFallbackToCustomNotificationAds};
+
+#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)
+
+// Ad notification fade animation duration in milliseconds
+constexpr base::FeatureParam<int> kCustomNotificationAdFadeDuration{
+    &kCustomNotificationAds, "ad_notification_fade_duration",
```

Comment:
now that we have cleaned up these features I think we can remove some of the verbosity from the name, i.e. "fade_duration", "dark_mode_background_color" etc.